src/test/java/org/opensearch/sdk/TransportCommunicationIT.java

```diff
@@ -145,7 +145,7 @@ public void run() {
     private void startTransportandClient(Settings settings, Thread client) throws IOException, InterruptedException {
 
         // retrieve transport service
-        ExtensionsRunner extensionsRunner = new ExtensionsRunner();
+        ExtensionsRunner extensionsRunner = new ExtensionsRunnerForTest();
```

Comment:
Should we use `ExtensionsRunnerForTest` for other occurrences in tests as well?
Like:
https://github.com/opensearch-project/opensearch-sdk-java/blob/main/src/test/java/org/opensearch/sdk/TestNamedWriteableRegistryAPI.java#L138

Reply:
@owaiskazi19 that particular reference is to the class which remains valid as a superclass. And that particular test is just choosing a class that we know is not a named writeable.  So I think that one is fine. Are there others?  The rest of the PR should have broken any no-arg constructors.